[2024-01-29T01:06:39.349+0100] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: match_id_dag.trigger_match_histories_dag manual__2024-01-29T00:06:17.070881+00:00 [queued]>
[2024-01-29T01:06:39.369+0100] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: match_id_dag.trigger_match_histories_dag manual__2024-01-29T00:06:17.070881+00:00 [queued]>
[2024-01-29T01:06:39.370+0100] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-29T01:06:39.409+0100] {taskinstance.py:2192} INFO - Executing <Task(TriggerDagRunOperator): trigger_match_histories_dag> on 2024-01-29 00:06:17.070881+00:00
[2024-01-29T01:06:39.413+0100] {standard_task_runner.py:60} INFO - Started process 21045 to run task
[2024-01-29T01:06:39.419+0100] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'match_id_dag', 'trigger_match_histories_dag', 'manual__2024-01-29T00:06:17.070881+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/match_id_dag.py', '--cfg-path', '/tmp/tmp8rmbrjyj']
[2024-01-29T01:06:39.423+0100] {standard_task_runner.py:88} INFO - Job 51: Subtask trigger_match_histories_dag
[2024-01-29T01:06:39.464+0100] {task_command.py:423} INFO - Running <TaskInstance: match_id_dag.trigger_match_histories_dag manual__2024-01-29T00:06:17.070881+00:00 [running]> on host ubuntu.ubuntu.virtualbox.org
[2024-01-29T01:06:39.535+0100] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='irachide1@gmail.com' AIRFLOW_CTX_DAG_OWNER='hadoop' AIRFLOW_CTX_DAG_ID='match_id_dag' AIRFLOW_CTX_TASK_ID='trigger_match_histories_dag' AIRFLOW_CTX_EXECUTION_DATE='2024-01-29T00:06:17.070881+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-29T00:06:17.070881+00:00'
[2024-01-29T01:06:39.621+0100] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=match_id_dag, task_id=trigger_match_histories_dag, execution_date=20240129T000617, start_date=20240129T000639, end_date=20240129T000639
[2024-01-29T01:06:39.692+0100] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-29T01:06:39.709+0100] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
